rpc/rpc-lib/src/rpc-clnt.c

```diff
@@ -322,7 +321,7 @@ saved_frames_unwind(struct saved_frames *saved_frames)
 
     list_for_each_entry_safe(trav, tmp, &saved_frames->sf.list, list)
     {
-        gf_time_fmt_tv(timestr, sizeof timestr, &trav->saved_at, gf_timefmt_FT);
+        gf_time_fmt(timestr, sizeof timestr, trav->saved_at, gf_timefmt_FT);
```

Comment:
nit: (unrelated to this change) - this line should come after lines 326-327

Reply:
Good catch indeed.